core/src/Threads/Kokkos_ThreadsTeam.hpp

```diff
@@ -679,8 +684,14 @@ class TeamPolicyInternal<Kokkos::Threads, Properties...>
   //----------------------------------------
 
   inline int team_size() const { return m_team_size; }
+  inline int vector_length() const { return 1; }
```

Comment:
same as before in OpenMP what does this mean? Do we really want this.

Reply:
If it's in the public interface, yes, it should be in all backends. My preference is to implement this as-is and fix the TeamPolicy API separately